### ISSUE TITLE
Hubble UI fails to render service map after helm deploy

### ISSUE THREAD

**Issue by itsthatriver** [cilium/cilium#18732]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I deployed Cilium in AWS (EKS) with helm (using the template from the below command).
```
helm template cilium cilium/cilium --version 1.11.0 \
  --namespace kube-system \
  --set ipam.mode=cluster-pool \
  --set tunnel=vxlan \
  --set localRedirectPolicy=true \
  --set egressMasqueradeInterfaces=eth+ \
  --set nodeinit.enabled=false \
  --set hubble.tls.auto.method="cronJob" \
  --set hubble.listenAddress=":4244" \
  --set hubble.relay.enabled=true \
  --set hubble.ui.enabled=true \
  --set upgradeCompatibility="1.9" \
  --set encryption.enabled=true \
  --set encryption.nodeEncryption=false \
  --set encryption.type=ipsec \
  --set prometheus.enabled=true \
  --set operator.prometheus.enabled=true \
  --set hubble.metrics.enabled="{dns,drop,tcp,flow,icmp,http}" \
  --set labels="k8s:io.kubernetes.pod.namespace k8s:k8s-app k8s:app k8s:name k8s:spark-role"
```

The entire cluster is healthy (we're using a private registry, but these are just copies of the images referenced by helm 1.11.0)
```
❯ cilium status
    /¯¯\
 /¯¯\__/¯¯\    Cilium:         OK
 \__/¯¯\__/    Operator:       OK
 /¯¯\__/¯¯\    Hubble:         OK
 \__/¯¯\__/    ClusterMesh:    disabled
    \__/

DaemonSet         cilium             Desired: 2, Ready: 2/2, Available: 2/2
Deployment        cilium-operator    Desired: 2, Ready: 2/2, Available: 2/2
Deployment        hubble-relay       Desired: 1, Ready: 1/1, Available: 1/1
Deployment        hubble-ui          Desired: 1, Ready: 1/1, Available: 1/1
Containers:       cilium             Running: 2
                  cilium-operator
                  hubble-relay       Running: 1
                  hubble-ui          Running: 1
Cluster Pods:     44/44 managed by Cilium
Image versions    hubble-relay    quay.io/ascendio/hubble-relay:v1.11.0: 1
                  hubble-ui       quay.io/ascendio/hubble-ui:v0.8.3: 1
                  hubble-ui       quay.io/ascendio/hubble-ui-backend:v0.8.3: 1
                  hubble-ui       quay.io/ascendio/envoy:v1.18.4: 1
                  cilium          quay.io/ascendio/cilium:v1.11.0: 2
```

I run `kubectl port-forward -n kube-system hubble-ui-5b7f99fcb6-5qqf2 :8080` and navigate to the output address in my browser and see this error
![image](https://user-images.githubusercontent.com/5632093/152935085-e1819fa3-f40e-489d-be0b-49e9bde8aae5.png)

I see some connection retries in hubble-relay (although they seem to stabilize after single retries). Neither hubble backend, frontend, nor proxy is producing any errors.

### Cilium Version

```
❯ cilium version
cilium-cli: v0.10.0 compiled with go1.17.4 on darwin/amd64
cilium image (default): v1.11.0
cilium image (stable): v1.11.1
cilium image (running): v1.11.0
```

### Kernel Version

Linux version 5.4.156-83.273.amzn2.x86_64 (mockbuild@ip-10-0-39-220) (gcc version 7.3.1 20180712 (Red Hat 7.3.1-13) (GCC)) #1 SMP Sat Oct 30 12:59:07 UTC 2021

### Kubernetes Version

```
❯ kubectl version
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.5", GitCommit:"aea7bbadd2fc0cd689de94a54e5b7b758869d691", GitTreeState:"clean", BuildDate:"2021-09-15T21:10:45Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"darwin/arm64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.5-eks-bc4871b", GitCommit:"5236faf39f1b7a7dabea8df12726f25608131aa9", GitTreeState:"clean", BuildDate:"2021-10-29T23:32:16Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
```

### Sysdump

[cilium-sysdump-20220207-230512.zip](https://github.com/cilium/cilium/files/8021529/cilium-sysdump-20220207-230512.zip)

### Relevant log output

```shell
# hubble relay
level=warning msg="Failed to create gRPC client" address="192.168.9.213:4244" error="connection error: desc = \"transport: error while dialing: dial tcp 192.168.9.213:4244: connect: connection refused\"" hubble-tls=true next-try-in=10s peer=ip-192-168-9-213.ec2.internal subsys=hubble-relay
level=info msg=Connecting address="192.168.9.213:4244" hubble-tls=true peer=ip-192-168-9-213.ec2.internal subsys=hubble-relay
level=info msg=Connected address="192.168.9.213:4244" hubble-tls=true peer=ip-192-168-9-213.ec2.internal subsys=hubble-relay

# hubble-ui backend
level=info msg="initialized with TLS disabled\n" subsys=config
level=info msg="listening at: 0.0.0.0:8090\n" subsys=ui-backend

# hubble-ui proxy
❯ kubectl logs -n kube-system hubble-ui-5b7f99fcb6-fx9j7 -c proxy | grep -v info
[2022-01-25 18:51:33.139][1][warning][main] [source/server/server.cc:506] No admin address given, so no admin HTTP server started.
  - name: base
    static_layer:
      {}
  - name: admin
    admin_layer:
      {}
[2022-01-25 18:51:33.151][1][warning][main] [source/server/server.cc:642] there is no configured limit to the number of allowed active connections. Set a limit via the runtime key overload.global_downstream_max_connections
```

### Anything else?

I'm seeing this same issue in EKS, AKS, and GKE, so I'm assuming that it's either a bug or an issue of my config (we're using very similar config in all 3 clouds)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by geakstr** [comment]
Hi @thejosephstevens! Could you try to port-forward to :80 port? `kubectl port-forward -n kube-system hubble-ui-5b7f99fcb6-5qqf2 :80`

**Comment by thejosephstevens** [comment]
> Hi @thejosephstevens! Could you try to port-forward to :80 port? kubectl port-forward -n kube-system hubble-ui-5b7f99fcb6-5qqf2 :80

So that doesn't work
```
❯ kubectl port-forward -n kube-system hubble-ui-5b7f99fcb6-ggfvl :80
Forwarding from 127.0.0.1:49652 -> 80
Forwarding from [::1]:49652 -> 80
Handling connection for 49652
E0208 11:07:30.135684   55786 portforward.go:400] an error occurred forwarding 49652 -> 80: error forwarding port 80 to pod 48e3f8b225b5eb1144574b4063b3dc89ee7d2a44ae11795bd27e3ab9cd621ab2, uid : failed to execute portforward in network namespace "/var/run/netns/cni-7af0be4c-347f-472f-aa2b-be6cd7aa9508": failed to dial 80: dial tcp4 127.0.0.1:80: connect: connection refused
```

but I got a response in slack and it turns out that using these commands works
```
cilium hubble port-forward&
cilium hubble ui
# browser opens to the hubble ui with all the namespaces correctly populated
```

So this will work fine for us and I'm happy to close the issue, but I guess I should ask, is that expected? Since I have this set up I'm happy to run tests if the original method I was using to port forward to the ui is supposed to work.

**Comment by thejosephstevens** [comment]
So I just took a look at the docs [here](https://docs.cilium.io/en/v1.11/gettingstarted/hubble/#open-the-hubble-ui) and it gives the right instructions to use the cilium cli to get to the hubble ui. I also got another note in Slack to use `kubectl -n kube-system port-forward svc/hubble-ui 12000:80`, which also works. I'll go ahead and close this since this appears to be a clear case of me failing to RTFM. If there's any follow-up questions feel free to ping me.
